downloader.sh

```diff
@@ -68,12 +74,16 @@ do
 		wget -o log.txt -O $i.html -c www.mangapanda.com/$manga/$chap/$i # Downloads the main webpage
 		grep 'src=\"http' $i.html | grep 'mangapanda' > jump.txt # Gets the list of image links
 		link=$(head -n 1 jump.txt)
-		starti="$(echo $link | grep -aob '"' | grep -oE '[0-9]+' | sed "11q;d")"
-		endi="$(echo $link | grep -aob '"' | grep -oE '[0-9]+' | sed "12q;d")"
+		starti=$(index link "11q")
```

Comment:
You are passing the arguments right.
It should be $link, so the function call becomes
```starti=$(index $link "11q")```
Apply it to all other places where it applies